docroot/sites/registrar.uiowa.edu/modules/registrar_core/src/Plugin/Block/CourseDeadlinesBlock.php

```diff
@@ -467,29 +556,49 @@ public function deadlinesMarkup($session, $department, $course, $section): array
         unset($dates['lastDayToAddDropNoFee']);
       }
 
+      $deadline_rows = [];
       foreach ($dates as $key => $label) {
         if ($data->$key) {
-          $items[] = [
-            '#markup' => $this->t('<span class="uiowa-maui-deadline-label">@label</span> <span class="uiowa-maui-deadline-date">@date</span>', [
-              '@label' => $label,
-              '@date' => date('m/d/Y', strtotime($data->$key)),
-            ]),
+          $deadline_rows[] = [
+            $label,
+            date('m/d/Y', strtotime($data->$key)),
           ];
         }
       }
 
-      $deadlines['deadlines_wrapper'] = [
+      $deadlines['deadlines_table'] = [
+        '#type' => 'table',
+        '#header' => [
+          $this->t('Deadline'),
+          $this->t('Date'),
+        ],
+        '#rows' => $deadline_rows,
+        '#attributes' => [
+          'class' => ['uiowa-maui-deadlines-table'],
+        ],
+        '#caption' => $this->t('Course Deadlines'),
+      ];
+
+      $contact_info = $this->t('<a href="@url">Collegiate Office Contact Information</a> to obtain collegiate approval.', [
+        '@url' => Url::fromUri('https://registrar.uiowa.edu/collegiate-office-contact-information-students')->toString(),
+      ]);
+
+      $deadlines['contact_wrapper'] = [
         '#type' => 'container',
-        '#suffix' => '<p style="text-align: center;"><a href="https://registrar.uiowa.edu/collegiate-office-contact-information-students">Collegiate Office Contact Information</a> to obtain collegiate approval.</p>',
         '#attributes' => [
-          'class' => ['uiowa-maui-course-deadlines-wrapper', 'deadlines-wrapper'],
+          'class' => [
+            'uiowa-maui-course-deadlines-wrapper',
+            'contact-wrapper',
+            'alert',
```

Comment:
Maybe this could be a `callout bg--gray` instead of alert here? You might need to load in the callout library as well.

Reply:
had to change type to html_tag to workaround the bg-- stripping in docroot/themes/custom/uids_base/templates/form/container.html.twig